OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -65,6 +65,17 @@ public class ArmamentInfo : UpgradableTraitInfo, IRulesetLoaded, Requires<Attack
 		public WeaponInfo WeaponInfo { get; private set; }
 		public WDist ModifiedRange { get; private set; }
 
+		public readonly Stance Targets = Stance.Enemy;
```

Comment:
`Targets` could be confused with target types. Perhaps `TargetAllegiances` or possibly just `Allegiances`.